setup.py

```diff
@@ -2,7 +2,37 @@
 
 import os
 import re
-from setuptools import setup, find_packages
+import sys
+import importlib
+from distutils.core import setup
+from setuptools import find_packages
+from setuptools.command.build_ext import build_ext
+from distutils.command.clean import clean
+
+# See https://gist.github.com/ctokheim/6c34dc1d672afca0676a for more details.
+try:
```

Comment:
Do we need this? We only now support Python 3.6+ so no need to test for `language_level = "3"`?